v4/source/teams.yaml

```diff
@@ -2392,7 +2392,9 @@
         - teams
       summary: Invite users to the team by email
       description: |
-        Invite users to the existing team usign the user's email.
+        Invite users to the existing team using the user's email.
+
+        The number of emails that can be sent is rate limited to 20 per hour with a burst of 20 emails. If the rate limit exceeds, the error message contains details on when to retry and when the timer will be reset.
```

Comment:
Shall we explain why the limit is `20`?

Reply:
It's set as a constant in the code. I don't _think_ there is a good reason. @crspeller added that. Probably he can explain if there was a reason behind it.

Reply:
I don't think there is much logic behind 20. Just need some limit.